nixos/modules/services/misc/moonraker.nix

```diff
@@ -0,0 +1,141 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.moonraker;
+  format = pkgs.formats.ini {
+    listToValue = l: "\n" + builtins.concatStringsSep "\n" (map (elem: "  ${elem}") l);
+    mkKeyValue = generators.mkKeyValueDefault {} ":";
+  };
+in {
+  options = {
+    services.moonraker = {
+      enable = mkEnableOption "Moonraker, an API web server for Klipper";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.moonraker;
+        description = "The Moonraker package.";
+      };
+
+      klipperSocket = mkOption {
+        type = types.path;
+        example = "/run/klipper/api";
+        description = "Path to Klipper's API socket.";
+      };
```

Comment:
Set a default

Reply:
Ok, will add a default to `services.klipper` as well.